renovate.json

```diff
@@ -3,5 +3,13 @@
     "config:js-lib",
     ":automergeMinor",
     ":automergePr"
+  ],
+  "packageRules": [
+    {
+      "packagePatterns": [
+        "@types/*"
+      ],
+      "rangeStrategy": "replace"
```

Comment:
`"replace"` seems to already be the default: https://renovatebot.com/docs/configuration-options/#rangestrategy